pages/quiz/about-you.tsx

```diff
@@ -99,17 +121,17 @@ export const AboutYouPage = (): JSX.Element => {
 
   return (
     <Layout>
-      <Section>
-        <HeaderComponent title="About you" />
-        <Navigation>
-          <LinkButton
-            textContent="back"
-            arrow="back"
-            onClick={() => router.back()}
-          />
-        </Navigation>
-        <QuestionSection>
-          <h2>How old are you?</h2>
+      <HeaderComponent title="Support in Lambeth" />
+      <Navigation>
+        <LinkButton
+          textContent="back"
+          arrow="back"
+          onClick={() => router.back()}
```

Comment:
This can be a link to the 'what-are-your-interests' page, I think I missed it in my last PR.